doc/source/ray-contribute/writing-code-snippets.rst

```diff
@@ -221,24 +221,10 @@ If your Python code is non-deterministic, or if your output is excessively long,
 Ignoring *doctest-style* outputs
 ================================
 
-To ignore parts of a *doctest-style* output, add `:options: +ELLIPSIS` to
-the `doctest` directive and replace problematic sections with ellipsis. ::
-
-    .. doctest::
-        :options: +ELLIPSIS
-
-        >>> import ray
-        >>> ray.data.read_images("example://image-datasets/simple")
-        Dataset(
-           num_blocks=...,
-           num_rows=...,
-           schema={image: numpy.ndarray(shape=(32, 32, 3), dtype=uint8)}
-        )
-
-If you omit the `doctest` directive, append `# doctest: +ELLIPSIS` to your code instead.
+To ignore parts of a *doctest-style* output, replace problematic sections with ellipsis. ::
```

Comment:
What should I do if I want to exact match `...`?

Reply:
You mean to literally match "..." and not match everything? You can disable it with `:options: -ELLIPSIS`, I think.

```
.. testcode::

    print("...")

.. testoutput::
    :options: -ELLIPSIS
    ...
```

Reply:
Yea, not sure whether people will ever want this but probably worth mentioning in the `writing-code-snippets.rst`

Reply:
@bveeramani is there is a decision for this?

Reply:
I don't think people will ever want to do this, so I'm not sure if it's worth documenting it. I want to keep the guide simple.

If people run into this issue later, I'll document this.